test/test_sparse.py

```diff
@@ -43,6 +43,7 @@
 class TestSparse(TestCase):
 
     def setUp(self):
+        TestCase.setUp(self)
```

Comment:
I wanted to do 
```
super().setUp()
```
but I don't think it played nice with the instantiation for device types

Reply:
Can you please add a bit longer comment here why it is not the case (as `super()` is right python3-way of calling the parent)

Reply:
Also, can we have some sort of a lint rule for that?